x/ibc-rate-limit/.gitignore

```diff
@@ -0,0 +1,21 @@
+# Generated by Cargo
+# will have compiled files and executables
+debug/
```

Comment:
What do you think about having this in the root git ignore to have a single source of truth for exclusions?

Reply:
Noticed that there is another `.gitignore` file. Is there a reason for that?

Reply:
I think it makes sense for per-contract .gitignore's

Reply:
The reason for all the gitignore files is that we use a template to generate the boilerplate on each contract, and the template comes with the gitignore. 

It makes sense to have only one source of truth. I'll merge those.

Reply:
merged. marking as resolved.